src/aiida/engine/processes/calcjobs/manager.py

```diff
@@ -152,9 +154,9 @@ async def _update_job_info(self) -> None:
             raise
         else:
             for job_id in self._polling_jobs:
-                future = self._job_update_requests.pop(job_id, None)
+                future = self._job_update_requests.pop(job_id, None)  # type: ignore[arg-type]
                 if future is None:
-                    self.logger.debug(
+                    self.logger.warning(  # type: ignore[unreachable]
```

Comment:
@khsrali I guess we misunderstood each other in the other PR. I was saying that I think this log message should have a higher severity so that it is printed even if the user did not set debugging options.